apps/dcos_l4lb/test/dcos_l4lb_mesos_poller_SUITE.erl

```diff
@@ -31,6 +31,7 @@ end_per_suite(Config) ->
 init_per_testcase(_, Config) ->
     meck:new(dcos_net_mesos_listener, [no_link]),
     meck:expect(dcos_net_mesos_listener, poll, fun meck_mesos_poll/0),
+    meck:expect(dcos_net_mesos_listener, init_metrics, fun () -> ok end),
```

Comment:
When I looked at the issue of init, I found it was already doing `meck:unload(dcos_net_mesos_listener)`, so I decided to add a dummy init_metrics function to the mock. The tests pass :tada:

Reply:
It's a band-aid, not a proper fix. Could you please fix the root cause? `r3:break()` might help you [1]

[1] https://www.rebar3.org/v3/docs/using-breakpoints-to-debug-tests